src/stare/cli/confnote.py

```
@@ -0,0 +1,192 @@
+
+@confnote_app.command("get")
+def confnote_get(
+    ref_code: Annotated[str, typer.Argument(help="CONF note temporary reference code")],
+    output_json: Annotated[
+        bool | None,
+        typer.Option(
+            "--json/--no-json",
+            help="Emit JSON. Default: auto (JSON when piped, Rich table when interactive).",
+        ),
+    ] = None,
+    no_cache: Annotated[
```

Ruff (0.15.11)

[warning] Boolean default positional argument in function definition

(FBT002)